spec/requests/webhooks_spec.rb

```diff
@@ -74,6 +76,28 @@
       create(:donation_detail, item: item, amount: 10_00)
     end
 
+    describe 'crawl receipts for donations outside february/lny crawl' do
+      let(:amount) { 5000 }
+      let(:item_type) { 'donation' }
+      let(:seller_id) { seller1.seller_id }
+
+      before do
+        Timecop.freeze(Date.new(2021, 3, 15))
```

Comment:
could be good to test edge cases at the beginning and end of month. I don't think we necessarily care if a few extra get in, but we do want to be careful not to reject at the wrong time -- might also make sure we cover @justintmckibben's point of UTC vs ET